lib/ansible/modules/system/iptables.py

```diff
@@ -238,7 +238,9 @@
     version_added: "2.1"
   uid_owner:
     description:
-      - Specifies the UID or username to use in match by owner rule.
+      - Specifies the UID or username to use in match by owner rule. When the
```

Comment:
Better formatting: `C(!) argument`

When we know what branches this fix will go into the description will need updating to say something along the lines of
`From Ansible 2.x when the C(!) argument is prepended....`

Reply:
@gundalow Not sure I understand the usage... will the `.x` be evaluated automatically ?

Reply:
oh, no. you will need to specify that (2.5 or 2.6)